roles/kubernetes-apps/container_runtimes/kata_containers/defaults/main.yaml

```diff
@@ -1,5 +1,5 @@
 ---
 
 kata_containers_qemu_overhead: true
-kata_containers_qemu_overhead_fixed_cpu: 10m
-kata_containers_qemu_overhead_fixed_memory: 290Mi
+kata_containers_qemu_overhead_fixed_cpu: 250m
```

Comment:
This seems a bit extreme compared to the old value.

Reply:
Nvm, just checked the upstream ones, thanks for linking it. It looks like our own values were too far off.